lib/quby/answers/services/updates_answers.rb

```diff
@@ -13,6 +13,7 @@ def initialize(answer)
 
         def update(new_attributes = {})
           answer.raw_params = new_attributes
```

Comment:
Daar heb ik naar gekeken, maar answer is niet altijd gegarandeerd een activerecord. Verder is het jammer dat die oplossing twee maal naar de db moet schrijven voor elke answer.